### PR TITLE
Ember: upgrade to 2.13

### PR DESCRIPTION
**Pull request by snewcomer**
modify some tests to use moduleForAcceptance as well.

### CODE REVIEW

**Comment by toranb** [comment]
Great work here @snewcomer !!!

File: tests/acceptance/arrays-test.js [hunk 1]
```diff
@@ -1,31 +1,27 @@
...
-    application = startApp();
-    store = application.__container__.lookup('service:simple-store');
+moduleForAcceptance('Acceptance: Arrays Test', {
+  beforeEach() {
+    this.store = this.application.__container__.lookup('service:simple-store');
   },
-  afterEach: function() {
-    Ember.run(application, 'destroy');
+  afterEach() {
+    delete this.store;
```
Comment (toranb) [inline tests/acceptance/arrays-test.js]: interesting - this equiv to doing? or did you throw that out

```js
this.store = null;
```

Comment (snewcomer) [reply tests/acceptance/arrays-test.js]: I think you are right.  I've read before delete may lead to memory leaks whereas null will always break references to the object.  So maybe safer to set to null.

**Comment by toranb** [comment]
@snewcomer thanks again for the effort here! I just published v5.6.0 w/ your changes!

**Pull request merged**